helix-term/src/commands.rs

```diff
@@ -5390,9 +5391,33 @@ fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
                 doc.set_selection(view.id, selection);
             };
             textobject(cx.editor);
+            cx.editor.autoinfo = None;
```

Comment:
Is this necessary?

Reply:
It seemed to be the pattern elsewhere, and seemed to be how to dismiss the info box. I'll try it without later.

Reply:
Confrimed, if I don't do this, the infobox remains until the next action occurs.